modules/install-sno-additional-requirements-for-installing-sno-on-a-cloud-provider.adoc

```diff
@@ -0,0 +1,18 @@
+// This module is included in the following assemblies: 
+//
+// installing/installing_sno/install-sno-preparing-to-install-sno.adoc
+
+:_content-type: CONCEPT
+[id="additional-requirements-for-installing-sno-on-a-cloud-provider_{context}"]
+= Additional requirements for installing {sno} on a cloud provider
+
+The documentation for installer-provisioned installation on cloud providers is written with a high availability cluster consisting of three control plane nodes. When referring to the documentation, consider the differences between the requirements for a {sno} cluster and a high availability cluster.
```

Comment:
Not sure why I don't like _is written with_ here.  It just sounds odd (like, this comment it written with a laptop).
```suggestion
The documentation for installer-provisioned installation on cloud providers is based on a high availability cluster consisting of three control plane nodes. When referring to the documentation, consider the differences between the requirements for a {sno} cluster and a high availability cluster.
```

Reply:
Note that this is copied from a different section of the docs, and is not new. It bothered me as well but I feel like this is irrelevant to this PR